frontend/src/pages/PassingTest/PassingTestContainer.tsx

```diff
@@ -141,18 +163,19 @@ const PassingTestContainer: React.FC = () => {
             resultEnabled={resultEnabled}
             stageNext={stageNext}
             stageBack={stageBack}
-            testItem={courseTest}
+            testTitle={courseTest?.title}
+            testDuration={testDuration}
             questionStageItem={questionStageItem}
             isLoading={courseTestResponseIsLoading}
             handleSubmitResult={handleSubmitResult}
             handleConfirm={handleConfirm}
             handleNavigateBack={handleNavigateBack}
           />
           <ConfirmLeavePage
-            isOpened={isConfirmOpen || showPrompt}
+            isOpened={isConfirmOpen || false}
             isLoading={courseTestResponseIsLoading}
-            handleCancelLeavePage={cancelNavigation}
-            handleLeavePage={confirmNavigation}
+            handleCancelLeavePage={() => {}}
+            handleLeavePage={() => {}}
```

Comment:
?

Reply:
problems with pr, its should be at another branch